Sparkle/SUPlainInstaller.m

```diff
@@ -20,8 +20,10 @@ + (void)performInstallationToPath:(NSString *)installationPath fromPath:(NSStrin
 
     // Prevent malicious downgrades
     if (![[[NSBundle bundleWithIdentifier:SUBundleIdentifier] infoDictionary][SUEnableAutomatedDowngradesKey] boolValue]) {
-        if ([comparator compareVersion:[host version] toVersion:[[NSBundle bundleWithPath:path] objectForInfoDictionaryKey:(__bridge NSString *)kCFBundleVersionKey]] == NSOrderedDescending) {
-            NSString *errorMessage = [NSString stringWithFormat:@"Sparkle Updater: Possible attack in progress! Attempting to \"upgrade\" from %@ to %@. Aborting update.", [host version], [[NSBundle bundleWithPath:path] objectForInfoDictionaryKey:(__bridge NSString *)kCFBundleVersionKey]];
+        // if ([comparator compareVersion:[host version] toVersion:[[NSBundle bundleWithPath:path] objectForInfoDictionaryKey:(__bridge NSString *)kCFBundleVersionKey]] == NSOrderedDescending) {
+        if ([comparator compareVersion:[host version] toVersion:[[NSBundle bundleWithPath:path] objectForInfoDictionaryKey:@"CFBundleShortVersionString"]] == NSOrderedDescending) {
+            // NSString *errorMessage = [NSString stringWithFormat:@"Sparkle Updater: Possible attack in progress! Attempting to \"upgrade\" from %@ to %@. Aborting update.", [host version], [[NSBundle bundleWithPath:path] objectForInfoDictionaryKey:(__bridge NSString *)kCFBundleVersionKey]];
```

Comment:
I'm not sure what our procedure is in these situations - I'm pretty sure you're commenting out instead of deleting for reference purposes in the future when updating Sparkle. 

I think it may be easier to review/understand later if the comment were "changing to use `CFBundleShortVersionString` instead of `kCFBundleVersionKey`" instead of having to figure out what changed with inspection. 

/minor-picking-nits